src/octopos.cpp

```diff
@@ -299,7 +298,7 @@ octopOS::~octopOS() {
 
 int octopOS::shmid = 0;
 int octopOS::tentacle_ids[NUMMODULES];
-std::vector<std::shared_ptr<tentacle>> octopOS::tentacles;
```

Comment:
Unsafe :(

Reply:
This whole branch is way behind master I think

Reply:
Rewrite it in rust

Reply:
+2